chemicaltoolbox/rdock/rdock_macros.xml

```diff
@@ -1,5 +1,5 @@
 <macros>
-    <token name="@TOOL_VERSION@">2013.1-0</token>
+    <token name="@TOOL_VERSION@">2013.1</token>
```

Comment:
Oh that is an interesting one ....

Reply:
Hm, do you know what the `-0` means here?